tests/appsec/iast/taint_sinks/test_sql_injection.py

```diff
@@ -74,6 +74,7 @@ def test_sql_injection(fixture_path, fixture_module, iast_context_defaults):
     line, hash_value = get_line_and_hash("test_sql_injection", VULN_SQL_INJECTION, filename=fixture_path)
     assert vulnerability["location"]["path"] == fixture_path
     assert vulnerability["location"]["line"] == line
```

Comment:
I miss some asserts in the sink points tests with:
```python
assert vulnerability["location"]["class"] == class_expected
assert vulnerability["location"]["function"] == function_expected
```

Reply:
:+1:

Reply:
Done. Added the method and class_name checks for any `location` check I could find in the codebase.